src/ConferenceItem.jsx

```diff
@@ -26,10 +26,19 @@ const ConferenceItem = ({ conference }) => {
   const year = date.getFullYear();
   const hours = date.getHours();
   const minutes = date.getMinutes();
+  const inputBackground = ({ hours }) => {
+    if (hours >= 18) {
+      return "#AD9FC8";
+    } else if (hours < 6) {
+      return "#AD9FC8";
+    } else {
```

Comment:
Strictly speaking since this is the default you could leave out the last else and just return :)

Reply:
We did wonder if the else was needed but I didn't think you could have an else if without an else, thanks we'll fix that.